suite/tests/client-interface/drwrap-drreg-test.dll.c

```diff
@@ -104,8 +108,8 @@ clean_call_rw(void)
     mc.flags = DR_MC_CONTROL | DR_MC_INTEGER;
     bool ok = dr_get_mcontext(drcontext, &mc);
     CHECK(ok, "dr_get_mcontext failed");
-    CHECK(IF_X86_ELSE(mc.xdx, mc.r1) == 4, "app reg val not restored for clean call");
-    IF_X86_ELSE(mc.xcx, mc.r2) = 3;
+    CHECK(IF_X86_ELSE(mc.xdx, IF_AARCHXX_ELSE(mc.r1, mc.a1)) == 4, "app reg val not restored for clean call");
```

Comment:
style: Line too long. This should fail the clang-format check.

Reply:
Fixed